### DISCUSSION TITLE
Cannot find "DtkBuildHelper" on building dtkcore

### DISCUSSION THREAD

**Discussion by mukif2** [linuxdeepin/developer-center#7030] [Development Questions | 开发答疑] [answered]
Hello guys, I'm trying to build dtkcore and cmake told me that it cannot find the module "DtkBuildHelper". You mentioned that the only dependency is QT5 and i did install it. Did I miss anything?

**Comment by mukif2** [comment]
> Hello guys, I'm trying to build dtkcore and cmake told me that it cannot find the module "DtkBuildHelper". You mentioned that the only dependency is QT5 and i did install it. Did I miss anything?

btw, I'm working on Loongnix20.5 with Loongson3A6000.

**Comment by BLumia** [comment, marked as answer]
DtkBuildHelper is a part of `dtkcommon`. Do you have that dependency installed? You can use `apt build-dep .` to help you install the required build-time dependencies.

**Reply by mukif2** [reply]
Thanks, though the build-dep command doesn't work well because of limited resources on loongarch, and I think have to build dependencies manually.  I'm nearly there.

**Reply by BLumia** [reply]
You can check the `debian` folder to get the full dependency list. By the way, there are existing SIG working on port DDE to other architectures that includes LoongArch. If you are interested, you can join them as well. Link: https://matrix.to/#/#deepin-ports:matrix.org :)
